great_expectations/data_context/data_context.py

```diff
@@ -3264,6 +3270,41 @@ def run_checkpoint(
             **kwargs,
         )
 
+    def add_profiler(
+        self,
+        name: str,
+        config_version: float,
+        rules: Dict[str, dict],
+        class_name: str,
+        module_name: Optional[str] = None,
+        variables: Optional[dict] = None,
+        commented_map: Optional[CommentedMap] = None,
+        ge_cloud_id: Optional[str] = None,
```

Comment:
@cdkini Are we going ahead with adding `ge_cloud_id`?  If so, then please do so consistently across all the relevant classes, just like in `Checkpoint` (after #4111 is merged, it will be easy to touch-up all involved code sections).  However, we definitely need to come up with a cleaner design.  Thanks!  /cc @donaldheppner

Reply:
I'm just working off of the spec - has this been discussed in detail? I know we're aiming to move away from this `ge_cloud_id` pattern but is this the place to draw the line?

Happy to make any necessary changes but we should discuss this live.

Reply:
we're going ahead with `ge_cloud_id` ... for now